tools/secret-rotation/Directory.Build.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <IsPackable>false</IsPackable>
```

Comment:
Do you plan to publish a package? If not how to you plan for folks to consume it?

Reply:
I assume we only need to make the entry point tool packable.

Reply:
right.  Only the CLI is packable.  I've added a ci.yml that packs it.

Reply:
It would be installed as a dotnet tool locally and invoked.  Probably through a script alongside the rotation config